shared/src/simulation/utils/patients-transported-promise.ts

```diff
@@ -0,0 +1,35 @@
+import { IsInt, IsUUID, Min } from 'class-validator';
+import { IsValue } from '../../utils/validators';
+import { getCreate } from '../../models/utils/get-create';
+import { UUID, uuidValidationOptions } from '../../utils';
+
+export class PatientsTransportPromise {
+    @IsValue('patientsTransportedPromise')
+    readonly type = 'patientsTransportedPromise';
```

Comment:
Please name the class and the type identifier consistently (besides capitalization).